Wikipedia/Code/NotificationsCenterViewModel.swift

```diff
@@ -162,6 +162,18 @@ final class NotificationsCenterViewModel: NSObject {
         }
     }
     
+    func markAllAsSeen() {
+        remoteNotificationsController.markAllAsSeen { result in
+            switch result {
+            case let .failure(error):
+                DDLogError("Error marking all notifications as seen: \(error)")
```

Comment:
You might want to skip this logging if the error is the new filters-enabled error you'll make, since it'll be a common path that's not actually a problem to need to know about.